### ISSUE TITLE
chart-diff: some diffs are hidden sometimes for branches pre chart model migration

### ISSUE THREAD

**Issue by lucasrodes** [owid/etl#3246]
### One-liner
After the recent migration to the `chart_config` table (see https://github.com/owid/owid-grapher/pull/3767), some chart diffs are hidden in the app.

### Context & details

Mojmir (from [Slack](https://owid.slack.com/archives/C0193RW5E2J/p1725544496100289)):

> It’s not easily fixable by code, but it’ll work if you re-save those charts in admin. (This will change their `updatedAt` timestamp and everything’s gonna work like before)

**Comment by Marigold** [comment]
For the context - I only saw this issue on one staging server, not sure if any other were affected. The problem was that function `_get_staging_creation_time` didn't return the correct time of server creation.

I haven't thoroughly investigated why. We use the creation time of `charts` table (with `show table status like 'charts'`), but it got changed on that server and charts that were updated before weren't showing up.
